VERSION

```diff
@@ -1 +1 @@
-17.0.2
+17.0.3
```

Comment:
Yeah I thought about this as I was making my initial commit...hence my PR comment 😆